diff --git a/Makefile b/Makefile
index 06c18fe..dfab5ac 100644
--- a/Makefile
+++ b/Makefile
@@ -37,10 +37,10 @@ ifeq ($(UNAME), Darwin)
 	@test -d $(DESTDIR)$(ERLDIR) || mkdir -p $(DESTDIR)$(ERLDIR)/$(EBIN_DIR)
 	@install -p -m 0644 $(APP_FILE) $(DESTDIR)$(ERLDIR)/$(APP_FILE)
 	@install -p -m 0644 $(ERL_OBJECTS) $(DESTDIR)$(ERLDIR)/$(EBIN_DIR)
-	@install -p -m 0644 priv/erlrtpproxy-ser.config $(DESTDIR)$(prefix)/etc/$(NAME).config
-#	@install -p -m 0755 priv/erlrtpproxy-ser.init $(DESTDIR)$(prefix)/etc/rc.d/init.d/$(NAME)
-	@install -p -m 0644 priv/erlrtpproxy-ser.sysconfig $(DESTDIR)$(prefix)/etc/$(NAME)
-	@install -d $(DESTDIR)$(prefix)/var/lib/$(NAME)
+	@install -p -m 0644 priv/erlrtpproxy-ser.config $(DESTDIR)/opt/local/etc/$(NAME).config
+#	@install -p -m 0755 priv/erlrtpproxy-ser.init $(DESTDIR)/opt/local/etc/rc.d/init.d/$(NAME)
+	@install -p -m 0644 priv/erlrtpproxy-ser.sysconfig $(DESTDIR)/opt/local/etc/$(NAME)
+	@install -d $(DESTDIR)/opt/local/var/lib/$(NAME)
 	@echo "$(NAME) installed. \n"
 else
 	install -D -p -m 0644 $(APP_FILE) $(DESTDIR)$(ERLDIR)/$(APP_FILE)
