diff --git a/Makefile b/Makefile
index c6cc2e5..77a0d22 100644
--- a/Makefile
+++ b/Makefile
@@ -26,12 +26,12 @@ ifeq ($(UNAME), Darwin)
 	@export
 	@install -p -m 0644 $(APP_FILE) $(DESTDIR)$(ERLDIR)/$(APP_FILE)
 	@install -p -m 0644 $(ERL_OBJECTS) $(DESTDIR)$(ERLDIR)/$(EBIN_DIR)
-	@install -p -m 0644 priv/erlrtpproxy.config $(DESTDIR)$(prefix)/etc/erl$(NAME).config
+	@install -p -m 0644 priv/erlrtpproxy.config $(DESTDIR)/opt/local/etc/erl$(NAME).config
 #	@install -p -m 0755 priv/erlrtpproxy.init $(DESTDIR)/etc/rc.d/init.d/erl$(NAME)
-	@install -p -m 0644 priv/erlrtpproxy.sysconfig $(DESTDIR)$(prefix)/etc/erl$(NAME)
-	@install -d $(DESTDIR)$(prefix)/var/lib/erl$(NAME)
-	@install -p -m 0644 priv/erlang.cookie $(DESTDIR)$(prefix)/var/lib/erl$(NAME)/.erlang.cookie
-	@install -p -m 0644 priv/hosts.erlang $(DESTDIR)$(prefix)/var/lib/erl$(NAME)/.hosts.erlang
+	@install -p -m 0644 priv/erlrtpproxy.sysconfig $(DESTDIR)/opt/local/etc/erl$(NAME)
+	@install -d $(DESTDIR)/opt/local/var/lib/erl$(NAME)
+	@install -p -m 0644 priv/erlang.cookie $(DESTDIR)/opt/local/var/lib/erl$(NAME)/.erlang.cookie
+	@install -p -m 0644 priv/hosts.erlang $(DESTDIR)/opt/local/var/lib/erl$(NAME)/.hosts.erlang
 	@echo "erl$(NAME) installed. \n"
 else
 	install -D -p -m 0644 $(APP_FILE) $(DESTDIR)$(ERLDIR)/$(APP_FILE)
