diff --git a/Makefile b/Makefile
index 7aac9cd..0de576d 100644
--- a/Makefile
+++ b/Makefile
@@ -20,10 +20,10 @@ compile:
 
 install: all
 	test -d $(DESTDIR)$(ERLDIR)/ebin || mkdir -p $(DESTDIR)$(ERLDIR)/ebin
-	test -d $(DESTDIR)$(prefix)/etc || mkdir -p $(DESTDIR)$(prefix)/etc
+	test -d $(DESTDIR)$(prefix)/opt/local/etc || mkdir -p $(DESTDIR)$(prefix)/opt/local/etc
 	install -p -m 0644 $(APP_FILE) $(DESTDIR)$(ERLDIR)/ebin
 	install -p -m 0644 $(ERL_OBJECTS) $(DESTDIR)$(ERLDIR)/ebin
-	install -p -m 0644 priv/erl$(NAME).config $(DESTDIR)/etc
+	install -p -m 0644 priv/erl$(NAME).config $(DESTDIR)/opt/local/etc
 
 test:
 	$(REBAR) eunit $(REBAR_FLAGS)
